packages/block-library/src/navigation/edit/index.js

```diff
@@ -231,23 +232,38 @@ function Navigation( {
 		clientId
 	);
 
+	const {
+		convert,
+		state: classicMenuConversionState,
+	} = useConvertClassicToBlockMenu( clientId );
+
+	const isConvertingClassicMenu =
```

Comment:
Maybe the exposed state should be "converted" rather than "fetching". It would be more descriptive and we could do away with this var.